concourse/pipelines/create-bosh-cloudfoundry.yml

```diff
@@ -1440,14 +1479,14 @@ jobs:
     plan:
       - aggregate:
           - get: pipeline-trigger
-            passed: [ 'cf-deploy']
+            passed: ['post-deploy','availability-tests']
```

Comment:
I guess post-deploy should depend on `avail tests` and then if this depends on post deploy, then it doesn't need to depend on `avail tests` again...